public/views/transaction/tx.html

```diff
@@ -1,68 +1,17 @@
-  <div class="col-xs-5">
-    <div class="ellipsis text-right">
-      <span class="text-muted">{{tx.timestamp | timeAgo }}</span>
-    </div>
-  </div>
-</div>
-<div class="row line-mid" data-ng-hide="!tx">
-  <div class="col-md-5">
-    <div class="row">
-      <div class="panel panel-default">
-        <div class="panel-body">
-          <div class="pull-right">{{tx.amount | currency:currency}} {{currency.symbol}}</div>
-          <div class="ellipsis">
-            <a href="/address/{{tx.senderId}}">{{tx | txSender}}</a>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="col-md-1">
-    <div class="hidden-xs hidden-sm text-center">
-    <span class="lead glyphicon glyphicon-arrow-right text-muted" data-ng-class="{
-      'text-danger': tx.type == 0 && $parent.address.address == tx.senderId && tx.senderId != tx.recipientId,
-      'text-success': tx.type == 0 && $parent.address.address && tx.senderId != $parent.address.address,
-      'text-muted': tx.type != 0
-    }"></span>
-    </div>
-    <div class="hidden-md hidden-lg text-center">
-      <span class="lead glyphicon glyphicon-arrow-down text-muted" data-ng-class="{
-        'text-danger': tx.type == 0 && $parent.address.address == tx.senderId && tx.senderId != tx.recipientId,
-        'text-success': tx.type == 0 && $parent.address.address && tx.senderId != $parent.address.address,
-        'text-muted': tx.type != 0
-      }"></span>
-    </div>
-  </div>
-  <div class="col-md-6">
-    <div class="row">
-      <div>
-        <div class="panel panel-default">
-          <div class="panel-body">
-            <div class="pull-right">{{tx.amount | currency:currency}} {{currency.symbol}}</div>
-            <div class="ellipsis" data-ng-show="tx.type == 0">
-              <a href="/address/{{tx.recipientId}}">{{tx | txRecipient}}</a>
-            </div>
-            <div class="ellipsis" data-ng-show="tx.type >= 1">{{tx | txType}}</div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-<div class="line-top row" data-ng-hide="!tx">
-  <div class="col-xs-12 col-sm-4 col-md-4">
-    <span class="txvalues txvalues-default">Fee: {{tx.fee | currency:currency}} {{currency.symbol}}</span>
-  </div>
-  <div class="col-xs-12 col-sm-8 col-md-8 text-right">
-    <span data-ng-if="block.confirmations > 0 || tx.confirmations > 0" class="txvalues txvalues-success">{{block.confirmations || tx.confirmations}} Confirmations</span>
-    <span data-ng-if="!block.confirmations && !tx.confirmations" class="txvalues txvalues-danger">Unconfirmed Transaction!</span>
-    <span class="txvalues txvalues-primary">{{tx.amount | currency:currency}} {{currency.symbol}}</span>
-  </div>
-</div>
+	<td>
+		<a class="txid" href="/tx/{{tx.id}}">{{tx.id}}</a>
+		<span class="btn-copy" clip-copy="tx.id"></span>
+	</td>
+	<td><a href="/address/{{tx.senderId}}">{{tx | txSender}}</a></td>
+	<td><a href="/address/{{tx.recipientId}}">{{tx | txRecipient}}</a></td>
```

Comment:
There should be no link when transaction type is special (for example `Delegate vote`).